node/Cargo.toml

```diff
@@ -22,7 +22,14 @@ sha3 = { version = "0.10" }
 anyhow = "1.0"
 async-trait = "0.1"
 tokio = { version = "1", features = ["full"] }
-tracing-subscriber = "0.2"
+tracing-subscriber = "0.3.16"
 async-channel = "1.7"
 
 rocksdb_lib = { package = "rocksdb", version = "0.19.0", default-features = false }
+dusk-wallet-core = "0.14.0-rc.0"
+dusk-bytes = "^0.1"
+
+[dev-dependencies]
+rand = "0.8.5"
+rand_core = "0.6.4"
```

Comment:
Change dependencies to use only major and minor version (unless a specific patch version is required)

Reply:
Done